src/bsp/src/stm32/hal/src/hal.c

```diff
@@ -32,6 +32,8 @@ void Hal_initMcu() {
 
     PHal_initMcu();
     UartPrint_init();
+
+    MX_USB_DEVICE_Init();
```

Comment:
Where was it initialized before?

Reply:
It seems like I had deleted it at some point